lib/defaults.js

```diff
@@ -18,6 +18,7 @@ export default function (self) {
       ['allowSorting', true],
       ['allowGroupingRows', true],
       ['allowGroupingColumns', true],
+      ['allowGridSizeChangeOnPaste', false],
```

Comment:
Proposal, but maybe a nitpick: `allowGridSizeChangeOnPaste` allows for the possibility that it might either expand _or_ contract, so maybe call it `allowGridExpandOnPaste` ?